starter-code/index.html

```diff
@@ -0,0 +1,245 @@
+                            </div>  
+                            <div class="item">
+                                    <img src="./img/slider-4.jpg" alt="slider-4">
+                                    <div class="carousel-caption">
+                                            <h3><b>ETIAM MAURIS TORTOR, PHARETRA QUIS LOBORTIS IN, PHARETRA IN DIAM</b></h3>
+                                            <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nec. Suspendisse at erat luctus; hendrerit sapien sed, consectetur erat. Maecenas dignissim suscipit orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse et libero nunc. Etiam mauris tortor, pharetra […]</p>
+                                            <button class="btn btn-primary" style="background-color:transparent">READ MORE</button>
+                                    </div>
+                            </div>                  
+                                                    
+                                                
+                          ...
+                        </div>
+                      
+                        <!-- Controls -->
+                        <a class="left carousel-control" href="#carousel-example-generic" role="button" data-slide="prev">
+                          <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+                          <span class="sr-only">Previous</span>
+                        </a>
+                        <a class="right carousel-control" href="#carousel-example-generic" role="button" data-slide="next">
+                          <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+                          <span class="sr-only">Next</span>
+                        </a>
+                </div>
+
+      </header>
+
+      <section class="services">
+            <div class="text-center container">
+                    <div class="col-md-4">
+                        <span><i class="glyphicon glyphicon-glass"></i></span>
+                        <h3>Web Design</h3>
+                        <p>fjqslmdkfjqsdmlkfj</p>
+                    </div>
+                    <div class="col-md-4">
+                            <span><i class="glyphicon glyphicon-star"></i></span>
+                            <h3>Web Development</h3>
+                            <p>fjqslmdkfjqsdmlkfj</p>
+                    </div>
+                    <div class="col-md-4">
+                            <span><i class="glyphicon glyphicon-heart"></i></span>
+                            <h3>User Interface</h3>
+                            <p>fjqslmdkfjqsdmlkfj</p>
+                    </div>
+                  </div>
+      </section>
+
+
+      <section class="container">
+          <article>
+              <h2>LATEST PROJECTS</h2>
```

Comment:
Looks like you tried to code all the page in one shot, but in my opinion it's not the best way, it's better to spend some times on the "Latest project" per exemple, as you will be able to reuse it for the next section..

This is basically why bootstrap is good, you can easily reuse your component :)